src/app/pages/datasets/components/roles-card/roles-card.component.component.html

```diff
@@ -0,0 +1,48 @@
+<mat-card class="card">
+  <mat-card-header>
+    <h3 mat-card-title>{{ 'Roles' | translate }}</h3>
+  </mat-card-header>
+  <mat-card-content>
+    <div class="details-item" *ngIf="dataset.apps?.length">
+      <mat-icon class="icon">apps</mat-icon>
+      <div class="label">{{ 'Apps' | translate }}:</div>
+      <div class="value">{{ 'This dataset is used by one or more apps' | translate }}</div>
+      <a
+        class="action"
+        [routerLink]="['/apps']"
+      >{{ 'Manage Apps Settings' | translate }}</a>
+    </div>
+    <div class="details-item" *ngIf="hasChildrenWithShares">
+      <mat-icon class="icon">share</mat-icon>
+      <div class="label">{{ 'Child Shares' | translate }}:</div>
+      <div class="value">{{ 'This dataset has children with shares' | translate }}</div>
+    </div>
+    <div class="details-item" *ngIf="dataset.smb_shares?.length">
+      <mat-icon svgIcon="ix:smb_share" class="icon"></mat-icon>
+      <div class="label">{{ 'Share Attached' | translate }}:</div>
+      <div class="value">{{ 'Dataset is shared via SMB as ' + smbSharesNames | translate }}</div>
```

Comment:
Better to use a placeholder instead of a concatenation, e.g. `'Dataset is shared via SMB as {shares}' | translate: { shares: smbSharesNames }`